packages/models/src/crypto-assets/bitcoin/inscription.model.ts

```diff
@@ -53,3 +56,103 @@ export function whenInscriptionMimeType<T>(
 
   throw new Error('Unhandled inscription type');
 }
+export interface Inscription extends InscriptionCryptoAssetInfo {
+  preview: string;
+  src: string;
+  title: string;
+  output: string;
+  txid: string;
+  offset: string;
+  address: string;
+  genesis_block_hash: string;
+  genesis_timestamp: number;
+  genesis_block_height: number;
+  value: string;
+}
+
+interface RawInscription {
+  id: string;
+  number: number;
+  output: string;
+  content_type: string;
+  tx_id: string;
+  offset: string;
+  address: string;
+  genesis_block_hash: string;
+  genesis_timestamp: number;
+  genesis_block_height: number;
+  value: string;
+}
+
+export function createInscription(inscription: RawInscription): Inscription {
```

Comment:
I see you use it here, but what if we want to create a `Inscription` type from another source? 

Two scenarios:
- What if we need to create an inscription from information passed in via an RPC call. Then we're having to create it by passing in parameters specific to the API (e.g. `tx_id` vs `txid`) etc
- Or, we use an additional API that has a `transactionId` prop name instead. Then we have to do some funky prop renaming: `createInscription({ tx_id: tx.transactionId, ...props })` when we really wanna stick to our `txid`.

Conceptually, I'm not sure @leather-wallet/models should "know" anything about API responses, and putting the response here means it does.

What do you think of doing the mapping of API response props → Inscription props all in the queries package? Where `createInscription` only really initialises the type?

Happy to jump on call and discuss live if I haven't explained this well

Reply:
I get that it's more convenient now, but if we can anticipate that it'll happen, let's avoid it, given it's just a matter of prop naming?

Reply:
Nvm here, I had missed the PR in the extension where more of this discussion happened.